package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redis-memory-server",
-  "version": "0.3.1",
+  "version": "0.4.0",
```

Comment:
please remove this change. i will bump the version as part of the release process.